monkey/monkey_island/cc/resources/configuration.py

```diff
@@ -0,0 +1,82 @@
+import json
+from dataclasses import dataclass
+from itertools import chain
+from typing import Mapping
+
+import marshmallow
+from flask import jsonify, request
+
+from common.configuration.agent_configuration import AgentConfigurationSchema
+from common.configuration.default_agent_configuration import DEFAULT_AGENT_CONFIGURATION
+from monkey_island.cc.repository import IAgentConfigurationRepository
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+from monkey_island.cc.resources.request_authentication import jwt_required
+
+
+class ImportStatuses:
+    INVALID_CONFIGURATION = "invalid_configuration"
+    IMPORTED = "imported"
+
+
+@dataclass
+class ResponseContents:
+    import_status: str = ImportStatuses.IMPORTED
+    message: str = ""
+    status_code: int = 200
```

Comment:
Why do we return import status?

1. The REST API doesn't provide an "import" endpoint, it provides a POST endpoint that allows a user to send data.
2. If the configuration is valid, we can just return a `200`
3. If the configuration is invalid, we should return the appropriate `4xx` response.

The `ImportStatuses` are completely redundant. The HTTP response codes are the status.

We probably don't need the `ResponseContents` class at all.